index.js

```diff
@@ -137,6 +138,19 @@ function report (times) {
   }))))
 }
 
+const argv = yargs
+  .usage('Usage: $0 [options]')
+
+  .boolean('production')
+  .describe('production', 'Skip devlopment dependencies (devDependencies)')
+
+  .boolean('no-optional')
+  .describe('no-optional', 'Skip optional dependencies (optionalDependencies)')
+
+  .help('help')
+  .alias('h', 'help')
+  .argv
```

Comment:
@nolanlawson I just checked, putting it at the top won't make a difference. `help` is by default shown at the end of parameters list. That's the standard thing to do.